modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita-bpm.adoc

```diff
@@ -36,18 +36,18 @@ You are recommended not to start 7.0.0 after you migrate to it, but to proceed i
 ====
 
 *RDBMS requirements:*
-The version targeted may not support the version of the database that is being migrated. You may then need to upgrade the version of your database prior to running the migration tool.
+The version targeted may not support the version of the database that is being migrated. You may then need to update the version of your database prior to running the migration tool.
```

Comment:
@vbotnari12 We should check if database vendors use the term update of upgrade of migrate for the Database server "mise à jour"...

Reply:
nice catch. AWS uses upgrade, Oracle uses both - upgrade and migration.